src/test/java/org/apache/commons/text/StringEscapeUtilsTest.java

```diff
@@ -49,12 +49,14 @@
     private static final String FOO = "foo";
 
     private static final String[][] HTML_ESCAPES = {
+            // message, expected, original
             {"no escaping", "plain text", "plain text"},
             {"no escaping", "plain text", "plain text"},
             {"empty string", "", ""},
             {"null", null, null},
             {"ampersand", "bread &amp; butter", "bread & butter"},
             {"quotes", "&quot;bread&quot; &amp; butter", "\"bread\" & butter"},
+            {"smart quotes", "&ldquo;bread and circuses&rdquo;", "\u201Cbread and circuses\u201d"},
```

Comment:
I see a new map with dozens of new entries but you are only testing a single value? Am I reading this right or are all the other map entries somehow also tested?

Reply:
I tested a single value because the tests seem to test only a few values; adding an exhaustive number of tests seems mostly just to reproduce the existing map